src/phoenix/trace/span_json_encoder.py

```diff
@@ -0,0 +1,63 @@
+import json
+from datetime import datetime
+from enum import Enum
+from typing import Any, List
+from uuid import UUID
+
+from .schemas import (
+    Span,
+    SpanContext,
+    SpanConversationAttributes,
+    SpanEvent,
+    SpanException,
+)
+
+
+class SpanJSONEncoder(json.JSONEncoder):
+    def default(self, obj: Any) -> Any:
```

Comment:
I think this is a good point about having a generic json encoder - the only reason I like the bespoke encoder here is that it can have some opinions about datetime etc. that we might not want in the generic one. I'm gonna leave this for now but with the note that we could move this into a util